WordPress/Classes/ViewRelated/NUX/SplashPrologueView.swift

```diff
@@ -0,0 +1,43 @@
+import SwiftUI
+
+struct SplashPrologueView: View {
+
+    var body: some View {
+        ZStack {
+            Color(SplashPrologueStyleGuide.backgroundColor)
+            HStack {
```

Comment:
A bit of a SwiftUI hack but this splits the two views, a rectangle and an image, 50% horizontally.